rpm-arm/Dockerfile

```
@@ -18,21 +16,31 @@ ENV DD_PIP_VERSION $DD_PIP_VERSION
 ENV DD_SETUPTOOLS_VERSION $DD_SETUPTOOLS_VERSION
 ENV CMAKE_VERSION $CMAKE_VERSION
 ENV CLANG_VERSION $CLANG_VERSION
-ENV GIMME_ARCH $TARGET_ARCH
+ENV DD_TARGET_ARCH $DD_TARGET_ARCH
 
 # configure yum and rpm for running on non-armv7l architectures
-RUN if [ "$TARGET_ARCH" = "arm32v7" ] ; then echo "armhfp" > /etc/yum/vars/basearch && \
+RUN echo "armhfp" > /etc/yum/vars/basearch && \
     echo "armv7hl" > /etc/yum/vars/arch && \
-    echo "armv7hl-redhat-linux-gnu" > /etc/rpm/platform ; fi
+    echo "armv7hl-redhat-linux-gnu" > /etc/rpm/platform
 
 # Resolves a nasty NOKEY warning that appears when using yum
-RUN if [ "$TARGET_ARCH" = "arm32v7" ] ; then rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-Arm32 ; fi
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-Arm32
 
 RUN yum groupinstall -y development && yum -y install which perl-ExtUtils-MakeMaker ncurses-compat-libs git procps \
     curl-devel expat-devel gettext-devel openssl-devel systemd-devel zlib-devel bzip2 glibc-static python-devel tar pkgconfig  \
     libtool autoconf policycoreutils-python
 
+# This trick preloads a dynamic library that will replace the uname syscall to pretend the machine is armv7l
+# The fakearmv7l library gets installed in "/usr/local/lib/libfakearmv7l.so"
+RUN git clone https://github.com/DataDog/fakearmv7l ; \
```

Doesn't seem likely that we will make changes to fakearmv7l but just in case we can do
```suggestion
RUN git clone --depth 1 https://github.com/DataDog/fakearmv7l ; \
```
so that if we ever make changes we don't clone the whole repo history